### PR TITLE
Added JWT grant type and upgraded the APIM version

### PR DESCRIPTION
**Pull request by ayyoob**

### CODE REVIEW

File: modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag [hunk 1]
```diff
@@ -7,829 +7,842 @@ var site = require("/site/conf/site.json");
...
-    };
-
-    var getTenantThemePath = function (){
-        return getTenantThemesPath() + getTheme().tenant_theme + "/";
-    }
+            };
+
+            var getThemeFile = function (path) {
+                var p, index, theme = getUserTheme();
+                if(theme.tenant_theme) {
```
Comment (prabathabey) [inline modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag:110]: Fix formatting issues in this particular line.

File: modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag [hunk 2]
```diff
@@ -7,829 +7,842 @@ var site = require("/site/conf/site.json");
...
+                    p = getTenantThemePath() + path;
+                    index = p.indexOf("?");
+                    if(new File(p.substring(0, index == -1 ? p.length : index)).isExists()) {
```
Comment (prabathabey) [inline modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag:113]: Same goes here.

File: modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag [hunk 3]
```diff
@@ -7,829 +7,842 @@ var site = require("/site/conf/site.json");
...
+                        return p;
+                    }
+                }
+                if(theme.subtheme) {
```
Comment (prabathabey) [inline modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag:117]: here as well.

Comment (ayyoob) [inline modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag:117]: Fixed with https://github.com/wso2/product-iots/pull/181/commits/c9736707cad38c5cafe01c90c363e91f93e74a18

File: modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag (hunk 2 above)
Comment (ayyoob) [inline modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag:113]: Fixed with https://github.com/wso2/product-iots/pull/181/commits/c9736707cad38c5cafe01c90c363e91f93e74a18

File: modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag (hunk 1 above)
Comment (ayyoob) [inline modules/distribution/src/repository/jaggeryapps/api-store/jagg.jag:110]: Fixed with https://github.com/wso2/product-iots/pull/181/commits/c9736707cad38c5cafe01c90c363e91f93e74a18

**Pull request merged**